VMBackup/main/Utils/EventLoggerUtil.py

```diff
@@ -0,0 +1,256 @@
+import os
+import threading
```

Comment:
do we not need compatibility check for it like we needed in Event.py

Reply:
so threading module is present in both the versions, just that getting the id of thread is not compatible.